### PR TITLE
Added GetUsersOptionTeamID so that a team ID can be passed in for use 

### PR DESCRIPTION
**Pull request by antonylucisano**
- GetUsers and GetUsersContext now takes Options
- Added team ID as an additional option as this is required for Organization level API calls used in GRID

### CODE REVIEW

File: users.go [hunk 1]
```diff
@@ -292,6 +292,13 @@ func GetUsersOptionPresence(n bool) GetUsersOption {
 	}
 }
 
+// GetUsersOptionTeamID include team_id
+func GetUsersOptionTeamID(team_id string) GetUsersOption {
```
Comment (kanata2) [inline users.go]: Prefer to use camel case in Go projects.

**Review by kanata2 — COMMENTED** [review]
Thanks!

File: users.go (hunk 1 above)
Comment (antonylucisano) [reply users.go]: Cool, have updated to camelCase

**Review by kanata2 — APPROVED** [review]

**Pull request merged**